app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,17 @@
+class LikesController < ApplicationController
+  def new
+    @like = Like.new
```

Comment:
- [ ] Here should be create method not the new method because new methods is used to handle the forms but for like you don't have any like so you should use create method here :blush: